sigstore-cli/build.gradle.kts

```diff
@@ -1,6 +1,9 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
 plugins {
     id("build-logic.java")
     id("application")
+    id("com.github.johnrengelman.shadow") version "8.1.1"
```

Comment:
https://github.com/GradleUp/shadow use this instead. 

I would try to look up if a repo has moved before adding a plugin/dependency. This has hit me in the past more than I would've liked.